src/app/Components/adminpanel/request-management/registration-confirm/registration-confirm.component.scss

```diff
@@ -0,0 +1,127 @@
+.feedback-card{
+    width: 80%;
+    min-width: 300px;
+    margin: 10px;
+    flex-wrap:  wrap;
+    align-content: flex-start;
+}
+
+.padding-left {
+  padding-left: 8px;
+}
+
+.padding-t-b {
+  padding-top: 5px 0;
+}
+
+.username{
+    line-height: 10px;
+    font-size: 30px;
+}
+
+.else-info{
+    font-size: 15px;
+}
+
+.margin-ten{
+    margin-left: 10px;
+    margin-right: 10px;
+}
+
+.margin-ten-full{
+  margin: 5px;
+}
+
+.status-buttons{
+    margin-left: -25px;
+}
+
+.minwidth{
+     min-width: 150px;
+}
+
+#navigate-div{
+    margin-top: 10px;
+}
+
+#component-box{
+    width: 100%;
+}
+
+#maintextbox{
+    text-align: justify;
+}
+
+.doc-pic{
+  width: 230px;
```

Comment:
with width and height of pictures it's better to use properties max-height, max-width, because of this they won't press and shakal, (But you decide)